docs/zh_cn/advanced_guides/customize_transforms.md

```diff
@@ -1,3 +1,237 @@
-# Customize Data Transformation and Augmentation
+# 自定义数据变换和数据增强
 
-Coming soon.
+### 数据变换
+
+在**OpenMMLab**算法库中，数据集的构建和数据的准备是相互解耦的，通常，数据集的构建只对数据集进行解析，记录每个样本的基本信息，而数据的准备则是通过一系列的数据变换，根据样本的基本信息进行数据加载、预处理、格式化等操作。
+
+### 数据变换的使用
+
+**MMPose**中的`数据变换`和`数据增强`类定义在[$MMPose/datasets/transforms](https://github.com/open-mmlab/mmpose/tree/dev-1.x/mmpose/datasets/transforms)目录中，对应的文件结构如下:
+
+```txt
+mmpose
+|----datasets
+    |----transforms
+        |----bottomup_transforms    # 自底向上
+        |----common_transforms      # 常用变换
+        |----converting             # 关键点转换
+        |----formatting             # 输入数据格式化
+        |----loading                # 原始数据加载
+        |----pose3d_transforms      # 三维变换
+        |----topdown_transforms     # 自顶向下
+```
+
+在**MMPose**中，**数据增强**和**数据变换**是使用者经常需要考虑的一个阶段，可参考如下流程进行相关阶段的设计：
+
+```mermaid
+flowchart LR
+  subgraph Transforms1
+  Flip
+  HalfBody
+  Rotation
+  Shift
+  Resize
+  ColorJittor
+  end
+  subgraph Transforms2
+  GenerateTarget
+  end
+  subgraph Transforms3
+  PackPoseInput
+  end
+  原始图片 --> Transforms1
+  原始标注 --> Transforms1
+  Transforms1 --> 输入图片
+  Transforms1 --> 输入坐标值
+  输入坐标值 --> Transforms2
+  Transforms2 --> Transforms3
+  输入图片 --> Transforms3
+
+```
```

Comment:
Hi @chaodyna , would you please save this diagram as an image and upload it? Our doc is based on readthedocs and it seems that the diagram cannot be rendered.

Reply:
You can check the compiled docs here:
![image](https://github.com/open-mmlab/mmpose/assets/13503330/8f184e52-ac65-426a-a518-ba1a72a9fbf7)

Reply:
Well, The flowchart is already in the assets and was wrote so well🤣. I changed the linking image to [mermaid.live](https://mermaid.live/), which is a useful tool and seems like steady to use. If there is any necessity to renovate and amend, I would like to learn from other friends and read the docs since it' been some time.